pom.xml

```diff
@@ -109,6 +109,7 @@
   <modules>
     <module>cxx-squid</module>
     <module>cxx-checks</module>
+    <module>CxxLint</module>    
```

Comment:
yep, i notice that :)

integrated into visual studio:

![image](https://cloud.githubusercontent.com/assets/1041352/13380004/18b0c30a-de3f-11e5-9f18-ef8eb98a768e.png)